Source/WebCore/animation/WebAnimation.cpp

```diff
@@ -301,20 +306,55 @@ void WebAnimation::effectTargetDidChange(const std::optional<const Styleable>& p
     InspectorInstrumentation::didChangeWebAnimationEffectTarget(*this);
 }
 
+#if ENABLE(CSS_TYPED_OM)
+ExceptionOr<std::optional<Seconds>> WebAnimation::validateCSSNumberishValue(std::optional<CSSNumberish> optionalCSSNumberish) const
```

Comment:
Should probably use `const std::optional<CSSNumberish>&` to avoid refcounting churn when CSSNumberish contains a RefPtr.

Reply:
Filed https://github.com/WebKit/WebKit/pull/5644 to fix a similar problem that had already snuck into `CSSScale`.